### PR TITLE
[PVR] [confluence] Use timer logos depending on its state.

### PR DESCRIPTION
**Pull request by Glenn-1990**
This is part 1 for:
https://github.com/xbmc/xbmc/pull/8657

1) This PR adds timer logos for disabled timers, conflicting timers and timers with an error.
2) Simplified the confluence part by using an include file.
3) Brackets of the recording icon are removed in order to have the same aspect ratio as all other timer icons.

schermafdruk van 2016-01-31 21 08 38

@ksooo @Jalle19 @ronie @metaron-uk

### CODE REVIEW

**Comment by ronie** [comment]
confluence changes look ok to me.

**Comment by Glenn-1990** [comment]
@ksooo reworked according comments

File: addons/skin.confluence/720p/includes.xml [hunk 1]
```diff
@@ -124,6 +124,23 @@
 		<value condition="!IsEmpty(Window.Property(IsRadio))">$LOCALIZE[19199] - $LOCALIZE[19024]</value>
 		<value>$LOCALIZE[19199] - $LOCALIZE[19023]</value>
 	</variable>
+  <variable name="PVRTimerIcon"> <!-- NOTE: this order is determing, 1st satisfying condition will be picked -->
```
Comment (phil65) [inline addons/skin.confluence/720p/includes.xml]: I dont think we need a comment to explain how $VARs work. ;)

**Comment by Glenn-1990** [comment]
@xhaggi Using your namings now.

**Comment by xhaggi** [comment]
thanks

**Comment by Glenn-1990** [comment]
@ksooo Good to go?

**Comment by ksooo** [comment]
jenkins build this please

**Comment by ksooo** [comment]
win-32 jenkins error is not related to this PR (sigh).

**Comment by xhaggi** [comment]
@Glenn-1990 it would be nice if you could also adjust our wiki infobool documentation or someone else.

**Comment by Glenn-1990** [comment]
@xhaggi will do

**Comment by da-anda** [comment]
thanks a lot. I really missed those after I switched from MP to Kodi for PVR

**Comment by MilhouseVH** [comment]
I'm guessing this PR is responsible for the following errors that are being logged since 7 Feb:
```
12:17:54  20.371561 T:1963352976    INFO: Loading skin file: DialogPVRGuideOSD.xml, load type: LOAD_ON_GUI_INIT
12:17:54  20.378744 T:1963352976   ERROR: Misplaced [
12:17:54  20.378815 T:1963352976   ERROR: Error parsing boolean expression IsEmpty($VAR[PVRTimerIcon])
12:17:54  20.379086 T:1963352976   ERROR: Misplaced [
12:17:54  20.379118 T:1963352976   ERROR: Error parsing boolean expression !IsEmpty($VAR[PVRTimerIcon])
```

Full debug log: http://sprunge.us/dEbY

**Comment by Glenn-1990** [comment]
@MilhouseVH will take a look, seems like a typo or something

**Comment by phil65** [comment]
@Glenn-1990 you cannot check vars with IsEmpty() afaik.

**Comment by Glenn-1990** [comment]
@MilhouseVH https://github.com/xbmc/xbmc/pull/9112

**Comment by MilhouseVH** [comment]
Thanks, errors have now gone.

**Pull request merged**